ee/dtr/release-notes.md

```diff
@@ -69,6 +73,10 @@ to upgrade your installation to the latest release.
   * Poll mirroring for Docker plugins such as `docker/imagefs` is currently broken. (docker/dhe-deploy #9490)
   * When viewing the details of a scanned image tag, the header may display a different vulnerability count from the layer details. (docker/dhe-deploy #9474)
   * In order to set a tag limit for pruning purposes, immutability must be turned off for a repository. This limitation is not clear in the **Repository Settings** view. (docker/dhe-deploy #9554)
+  * Changing your S3 settings through the web interface will lead to erased metadata. (ENGDTR-793)
+
+* CLI
+  * When reconfiguring and restoring DTR, specifying `--nfs-storage-url` will assume you are switching to a fresh storage backend and will wipe your tag and event history. (ENGDTR-794)
```

Comment:
Turns out it's just the tags that get wiped. Repo event history is still there.

Reply:
Nice - thank you! I'll keep it consistent too.